examples/custom-relu-mnist.jl

```diff
@@ -0,0 +1,102 @@
+using Statistics
+using DiffOpt
+using Flux
+using Flux: onehotbatch, onecold, crossentropy, throttle
+using Base.Iterators: repeated
+using OSQP
+using JuMP
+using ChainRulesCore
+
+## prepare data
+imgs = Flux.Data.MNIST.images()
+labels = Flux.Data.MNIST.labels();
+
+# Preprocessing
+X = hcat(float.(reshape.(imgs, :))...) #stack all the images
+Y = onehotbatch(labels, 0:9); # just a common way to encode categorical variables
+
+test_X = hcat(float.(reshape.(Flux.Data.MNIST.images(:test), :))...)
+test_Y = onehotbatch(Flux.Data.MNIST.labels(:test), 0:9)
+
+# float64 to float16, to save memory
+X = convert(Array{Float16,2}, X) 
+test_X = convert(Array{Float16,2}, test_X)
+
+X = X[:, 1:1000]
+Y = Y[:, 1:1000];
+
+"""
+    relu method for a Matrix
+"""
+function myRelu(y::AbstractMatrix{T}; model = Model(() -> diff_optimizer(OSQP.Optimizer))) where {T}
+    x̂ = zero(y)
+    
+    # model init
+    N = length(y[:, 1])
+    empty!(model)
+    set_silent(model)
+    @variable(model, x[1:N] >= 0)
+    
+    for i in 1:size(y, 2)
+        @objective(
+            model,
+            Min,
+            x'x -2x'y[:, i]
```

Comment:
This is where the warning comes from I believe, this should be fixed by https://github.com/jump-dev/MutableArithmetics.jl/pull/87

Reply:
solved. seems there's a `set_objective_coefficient` method. + speeds up training a bit